### ISSUE TITLE
Don't show Content-Encoding

### ISSUE THREAD

**Issue by Goooler** [ChuckerTeam/chucker#608]
## :writing_hand: Describe the bug

Don't show Content-Encoding in this case.

## :bomb: Steps to reproduce

1. Add my `HeaderInterceptor`
2. Add `ChuckerInterceptor.Builder(context).build()`
3. Did not show `Content-Encoding` in Chucker, but showed in Charles

## :camera: Screenshots
<div align="center">
<img src="https://user-images.githubusercontent.com/10363352/114535397-95552980-9c82-11eb-94c8-7ed3ae3ada96.jpg" height="300px" alt="图片说明" >
<img src="https://user-images.githubusercontent.com/10363352/114535427-a00fbe80-9c82-11eb-93cb-af4933b1506d.jpg" height="300px" alt="图片说明" >
<img src="https://user-images.githubusercontent.com/10363352/114535439-a30aaf00-9c82-11eb-8a69-19a64b70c058.png" height="300px" alt="图片说明" >
</div>

## :iphone: Tech info
 - Device: OnePlus 8t
 - OS: 11
 - Chucker version: 3.4.0

**Comment by MiSikora** [comment]
How do you add `ChuckerInterceptor` to `OkHttp`? As a network or application interceptor?

**Comment by Goooler** [comment]
> How do you add `ChuckerInterceptor` to `OkHttp`? As a network or application interceptor? If as an application interceptor then this is expected.

![image](https://user-images.githubusercontent.com/10363352/114734780-1db4f680-9d77-11eb-802b-068b615127b2.png)

**Comment by MiSikora** [comment]
Because this is how `OkHttp` [works](https://square.github.io/okhttp/interceptors/). If you want to inspect network traffic apply `ChuckerInterceptor` as a network interceptor.

One thing we could do, because I see a lot of confusion about this is to display both network and application headers. But this is a topic for a different discussion. I wonder what other maintainers think of such an addition.

**Comment by Goooler** [comment]
@MiSikora Thx, I got it.

**Comment by vbuberen** [comment]
I don't think we need to show both types of headers. I would just update README to explain the topic better.

**Comment by cortinico** [comment]
> One thing we could do, because I see a lot of confusion about this, is to display both network and application headers if it is applied as an application interceptor. But this is a topic for a different discussion. I wonder what other maintainers think of such an addition.

To be honest I'm a bit unsure on this point. On one hand, I think that showing all the Headers we can provide to the user will reduce frictions (i.e. probably a user just want to see the HTTP headers without having to know at which stage of the chain a header was added). At the same time, I'm afraid this could create more confusion for users that are actually aware of the network/application interceptors differences 🤔

**Comment by MiSikora** [comment]
I meant to show it in two sections. Network section and application section. Personally, I wouldn't bother, but I see that it comes up once in a while like in this issue so I just threw an idea.

**Comment by cortinico** [comment]
> I meant to show it in two sections. Network section and application section.

+1 on having them separated
